after/queries/typescript/highlights.scm

```diff
@@ -0,0 +1,12 @@
+;; extends
+[
+("=>" @operator.fat_arrow (#set! "priority" 1000))
+]
+
```

Comment:
why the set priorities everywhere? Precedence (currently..might change soon since we're trying to collaborate with other editors/upstream tree-sitter) is determined by the order of the nodes, and this node will be matched **after** the base queries as it's in the after/ directory I believe, so no need for these priorities

Reply:
This was probably a mistake on my part. The theme uses a somewhat complex caching system which I hadn't disabled so these weren't showing up. Good point on them being in the after directory 😆 .